docs/getting-started.md

```diff
@@ -84,12 +81,12 @@ The third step in creating this app is to create a controller to handle the rout
 In that file, you'll need this content:
 
 ```js
-exports.index = function (req) {
-  this.render({view: true, locals: {ip: req.connection.remoteAddress, ua: req.headers['user-agent']}});
+exports.index = function (req, render) {
+  render({view: true, locals: {ip: req.connection.remoteAddress, ua: req.headers['user-agent']}});
 };
 
-exports.json = function (req) {
-  this.render({json: {status: 'up', ip: req.connection.remoteAddress, ua: req.headers['user-agent']}});
```

Comment:
This form is correct for 0.3.0 - no `render` parameter for controller actions, and `render` prefixed with `this.`